src/drone_controller/input_layer/drone_state.py

```diff
@@ -0,0 +1,72 @@
+# pylint: disable=too-many-arguments
+# a wrapper class needs more parameter
+from src.drone_controller.exception.exceptions import DroneControllerError
+
+
+class DroneState:
+    """
+    Wrapper class for the state of the drone.
+    """
+
+    def __init__(self, position, velocity, velocity_ang, acceleration, acceleration_ang):
+        self._position = position
+        self._velocity = velocity
+        self._velocity_ang = velocity_ang
+        self._acceleration = acceleration
+        self._acceleration_ang = acceleration_ang
+
+    @property
+    def state(self):
+        """
+        Maps the drone state to a dictionary.
+        :param self: this drone state
+        :return: dictionary
+        """
+        return {
+            "Position": self._position,
+            "Velocity": self._velocity,
+            "Angular Velocity": self._velocity_ang,
+            "Acceleration": self._acceleration,
+            "Angular Acceleration": self._acceleration_ang
+        }
+
+    @classmethod
+    def create(cls, position, velocity, velocity_ang, acceleration, acceleration_ang):
```

Comment:
Conventionally the \_\_init\_\_ method is used to create a class.
I see that you want to try out classmethods, however with this you are programming a confusing redundancy.

Reply:
yes that's true will change that